ethcore/light/src/on_demand/mod.rs

```diff
@@ -29,7 +29,17 @@ use futures::sync::oneshot::{self, Sender, Receiver, Canceled};
 use network::PeerId;
 use util::{RwLock, Mutex};
 
-use net::{self, Handler, Status, Capabilities, Announcement, EventContext, BasicContext, ReqId};
+use net::{
```

Comment:
better to keep it on a few, indented lines as here: https://github.com/paritytech/parity/pull/5958/files#diff-e8c26f6188b2b873a7467abd3041b843R42